shared-scripts.sh

```diff
@@ -275,3 +272,25 @@ function _scanArtifacts {
         exit 1
     fi
 }
+function _cleanupResources {
+    #restore cache
+    loadCache repo $CODEBUILD_SRC_DIR
+    loadCache .cache $HOME/.cache
+    
+    #loading test account credentials
+    _loadTestAccountCredentials
+
+    cd packages/amplify-e2e-tests
+    yarn clean-e2e-resources
```

Comment:
add-circleci-tags.ts, and cleanup-e2e-resources.ts will need to be updated for this to work; as it is dependent on the circleci-api. A new method of tagging resources should be implemented with Codebuild in mind